src/collection/ngrx/ngrx-schema.json

```diff
@@ -55,9 +55,9 @@
       "default": false,
       "description": "Do not add NgRx dependencies to package.json (e.g., --skipPackageJson)"
     },
-    "creator": {
+    "creators": {
       "type": "boolean",
-      "default": true,
+      "default": false,
```

Comment:
There should still be the true. We would like treat the new syntax as default one.